docs/AR.md

```diff
@@ -0,0 +1,131 @@
+
+## Customisation of the Augmented Reality scene
+
+**Note!** To view the customisation in Augmented Reality scene, you need to delete the 3D model you downloaded in the application and download it again!
+
+- **Name:** Write the name you want your 3D model to be displayed in this section (Required)
+- **Description:** Write the description you want your 3D model to be displayed in this section (Optional)
+- **Geo-Koordinaten:**
+  - **Latitude & Longitude:** Write the coordinates of your 3D model that you want to be displayed on the map in this section (Optional)
+- **Animation name:** The field where the name of the animation of the 3D model can be written (Optional)
+- **Package size:** This is the field where the total size of all Augmented Reality scene objects you have loaded should be written in bytes. The value added to this section allows the user to show how many MB files will be downloaded in the application. (Required)
+- **VRX & MP4 & IMAGE:**
+  - Set the Position, Size and Rotation properties according to x,y,z coordinates. [Scenes](https://viro-community.readme.io/docs/scenes)
+- **TARGET:**
+  - _Physical width_ - Allows you to set the distance in the physical world (in metres) between the image and the device's camera.
+- **MP3:**
+  - _Is spatial sound_ - A field that allows you to specify whether the audio file you are uploading has spatial audio capability. takes true or false value [ViroSpatialSound](https://viro-community.readme.io/docs/virospatialsound#onfinish)
+  - If Spatial Audio is true:
+    - _Max distance:_ A distance after which the audio can not be heard.
+    - _Min distance:_ A distance after which the audio will begin to attenuate until maxDistance where the sound is completely gone.
+    - _Rolloff model:_ The rolloff model which determines how the sound volume will fall off between minDistance and maxDistance. Can take `None`, `Linear` & `Logarithmic` values as String
+- **MP4:**
+  - _Chroma key filtered video_ - If the video you want to add to the Augmented Reality scene has a background colour that you want to delete, add the colour code in HEX format here.
+
+## Display of added tourstopps in the app
+
+To download the 3D Model from the Tour detail screen:
+
+1. Press the hamburger menu button in the top right corner of the app and select "Tours and Places" from the drop-down menu,
+2. Scroll to the bottom of the page and in the "Tours" category select the section where you have added Tourstopps,
+3. Select the tour you added Tourstopps to from the displayed tour list and scroll to the bottom of the tour details page,
+4. Select the downloaded file from the Artworks list and click on the "Downloaden & AR Kunst gucken" button and wait for the 3D object to download,
+5. Click the View AR Art button and view the 3D model on the screen that opens.
```

Comment:
there is no "View AR Art" button on the screen right, because German. let's say "when finished, click the button to view the 3D model on the screen that is navigated to"?

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/659/commits/e29043d4a78964b955874ece2ce58b818c237f9b